diff -urN plugin-gargoyle-tinyproxy//files/www/js/tinyproxy.js plugin-gargoyle-tinyproxy-orig//files/www/js/tinyproxy.js
--- plugin-gargoyle-tinyproxy//files/www/js/tinyproxy.js	2010-11-27 10:05:31.201880323 +0100
+++ plugin-gargoyle-tinyproxy-orig//files/www/js/tinyproxy.js	2010-11-27 10:02:59.000000000 +0100
@@ -18,11 +18,11 @@
 	var errors = [];
 	if (tinyproxyPort == "")
 	{
-		errors.push("Wpisz port tinyproxy");
+		errors.push("Enter tinyproxy port");
 	}
 	if (errors.length > 0)
 	{
-		alert( errors.join("\n") + "\n\nNie można zapisać ustawień" );
+		alert( errors.join("\n") + "\n\nCould not save settings" );
 		return;
 	}
 	
diff -urN plugin-gargoyle-tinyproxy//files/www/tinyproxy.sh plugin-gargoyle-tinyproxy-orig//files/www/tinyproxy.sh
--- plugin-gargoyle-tinyproxy//files/www/tinyproxy.sh	2010-11-26 15:39:25.344908334 +0100
+++ plugin-gargoyle-tinyproxy-orig//files/www/tinyproxy.sh	2010-11-25 22:19:15.000000000 +0100
@@ -23,35 +23,35 @@
 		<div id='tinyproxy_enabled_container' class='rightcolumn'>
 		<label id='tinyproxy_enabled_label' class='leftcolumn'>Tinyproxy</label>
 			<select class="rightcolumn" id="tinyproxy_enabled" onchange="setVisibilityTinyproxy()">
-				<option value="1">Włączone</option>
-				<option value="0">Wyłączone</option>
+				<option value="1">Enabled</option>
+				<option value="0">Disabled</option>
 			</select>
 		</div>
 
 	<div id="tinyproxy_container">
 		<div id='tinyproxy_transp_container'>
-		   <label id="tinyproxy_transp_label" class="leftcolumn" for="tinyproxy_transparent">Przezroczyste proxy:</label>
+		   <label id="tinyproxy_transp_label" class="leftcolumn" for="tinyproxy_transparent">Transparent proxy:</label>
 		      <select class="rightcolumn" id="tinyproxy_transparent">
-                                <option value="1">Włączone</option>
-                                <option value="0">Wyłączone</option>
+                                <option value="1">Enabled</option>
+                                <option value="0">Disabled</option>
               </select>
 		</div>
 		
         <div id='tinyproxy_filterurl_container' class='rightcolumn'>
-            <label id='tinyproxy_filterurl_label' class='leftcolumn'>Filtrowanie treści:</label>
+            <label id='tinyproxy_filterurl_label' class='leftcolumn'>Filter URL</label>
                <select class="rightcolumn" id="tinyproxy_filterurl" onchange="setVisibilityFilterURL()">
-                                <option value="On">Włączone</option>
-                                <option value="Off">Wyłączone</option>
+                                <option value="On">Enabled</option>
+                                <option value="Off">Disabled</option>
                </select>
         </div>
 		
 		<div id='tinyproxy_filterurledit_container'>
-                        <label id="tinyproxy_filterurledit_label" class="leftcolumn" for="tinyproxy_filterurledit">Zawartość filtra:</label>
+                        <label id="tinyproxy_filterurledit_label" class="leftcolumn" for="tinyproxy_filterurledit">Filter URL Content:</label>
                         <textarea id="tinyproxy_filterurledit" class="nocolumn" rows='4' cols='30'/></textarea>
                 </div>
 
                 <div id='tinyproxy_startservers_container'>
-		<label id='tinyproxy_startservers_label' class='leftcolumn'>Ilość procesów programu:</label>
+		<label id='tinyproxy_startservers_label' class='leftcolumn'>Start Servers</label>
                         <select class="rightcolumn" id="tinyproxy_startservers">
                                 <option value="1">1</option>
                                 <option value="2">2</option>
@@ -68,43 +68,41 @@
                 </div>
 
 		<div id='tinyproxy_port_container'>
-			<label id="tinyproxy_port_label" class="leftcolumn" for="tinyproxy_port">Port serwera proxy:</label>
+			<label id="tinyproxy_port_label" class="leftcolumn" for="tinyproxy_port">Proxy port:</label>
                 	<input id="tinyproxy_port" class="rightcolumn" type=text" size='5'/>
 		</div>
 
                 <div id='tinyproxy_allowips_container'>
-                        <label id="tinyproxy_allowips_label" class="leftcolumn" for="tinyproxy_allowips">Dozwolone adresy IP:</label>
+                        <label id="tinyproxy_allowips_label" class="leftcolumn" for="tinyproxy_allowips">Allow IPs:</label>
                         <input id="tinyproxy_allowips" class="rightcolumn" type=text" size='17'/>
                 </div>
 
 		<div id='tinyproxy_loglevel_container'>
-                        <label id="tinyproxy_loglevel_label" class="leftcolumn" for="tinyproxy_port">Poziom logowania:</label>
+                        <label id="tinyproxy_loglevel_label" class="leftcolumn" for="tinyproxy_port">Log level:</label>
                         <select class="rightcolumn" id="tinyproxy_loglevel">
-                                <option value="Critical">Tylko krytyczne</option>
-                                <option value="Error">Błędy programu</option>
-				<option value="Warning">Ostrzeżenia programu</option>
-				<option value="Notice">Informacje programu</option>
-				<option value="Connect">Nawiązanie połączenia</option>
-				<option value="Info">Wszystko</option>
+                                <option value="Critical">Critical</option>
+                                <option value="Error">Error</option>
+				<option value="Warning">Warning</option>
+				<option value="Notice">Notice</option>
+				<option value="Connect">Connect</option>
+				<option value="Info">Info</option>
                 </select>
 		</div>
 
 		<div class='rightcolum' style="margin-bottom:15px">
-			<input type='button' id="tinyproxy_logview_button" value="Pokaż log" class="default_button" onclick="showLog()" />
+			<input type='button' id="tinyproxy_logview_button" value="Show Log" class="default_button" onclick="showLog()" />
 		</div>
         <div>
-             <em>UWAGA: Uważaj ze zbyt dużą ilością procesów programu na routerach z małą ilością pamięci RAM. Zalecane wartości to 
-		2-3 procesy. Poziom logowania: Najmniej szczegółowym poziomem logowania jest 'Tylko krytyczne', najbardziej 
-		szczegółowym - 'Wszystko'. Wybierając poziom logowania 'Informacje programu', w logu znajdą się też 'Uwagi programu', 
-		'Błędy programu' i 'Tylko krytyczne'. Poziom 'Nawiązane połączenia' zapisuje do pliku wszystkie nawiązane połączenia 
-		przez maszyny działające za proxy.</em>
+             <em>WARNING: Be careful with too large value of the Start Servers on routers with a small amount of RAM memory.
+             The default value is 10. The recommended process is 2-3.
+             Log Level: Critical (least verbose), Connect (log connections without Info's noise), Info (most verbose).</em>
         </div>
 
 	</div>
 
 <div id="bottom_button_container">
-	<input type='button' value='Zapisz zmiany' id="save_button" class="bottom_button" onclick='saveChanges()' />
-	<input type='button' value='Anuluj' id="reset_button" class="bottom_button" onclick='resetData()'/>
+	<input type='button' value='Save Changes' id="save_button" class="bottom_button" onclick='saveChanges()' />
+	<input type='button' value='Reset' id="reset_button" class="bottom_button" onclick='resetData()'/>
 </div>
 
 
